proposal.html

```diff
@@ -543,3 +543,20 @@
     </emu-grammar>
   </ins>
 </emu-clause>
+
+
+<!-- es6num="9.2.7.1" -->
+<emu-clause id="sec-%throwtypeerror%">
+  <h1>(<a href="#sec-%throwtypeerror%">9.2.7.1</a>) %ThrowTypeError% ( )</h1>
+  <p>The <dfn>%ThrowTypeError%</dfn> intrinsic is an anonymous built-in function object that is defined once for each realm. When %ThrowTypeError% is called it performs the following steps:</p>
```

Comment:
That's not something I have written; it is copied from the specification. Please only review my contributions, lines 556-558.